repo/lammps/package.py

```diff
@@ -9,10 +9,22 @@
 
 class Lammps(BuiltinLammps):
 
+  depends_on("kokkos+openmp", when="+openmp")
```

Comment:
Is this not in the upstreamed package?  Did ORNL not need this?

Reply:
I think there is a standalone openmp mode in lammps where we disable the kokkos pacakge and just enable the openmp one (~kokkkos+openmp-package) With this spack spec, we always enable kokkos with the desired backend (+kossos+openmp)
We can ask ORNL which one of `+kokkos+openmp` or `+openmp-package+openmp` is preferable.